lib/src/components/common_components.dart

```diff
@@ -29,14 +29,8 @@ class CalendarPageHeader extends StatelessWidget {
   /// Provides string to display as title.
   final StringProvider dateStringBuilder;
 
-  /// backgeound color of header.
```

Comment:
@faiyaz-shaikh  Don't remove existing parameters. For now we can deprecate these three parameters and then we can remove this in next major release.

Reply:
Consider this for all the public widgets/class that this plugin is providing.